webroot/src/components/Forms/_mixins/form.mixin.ts

```diff
@@ -211,6 +220,47 @@ class MixinForm extends Vue {
         this.updateFormSubmitError(errorMessage);
     }
 
+    showInvalidFormError(): void {
+        const { formData } = this;
+
+        // Find the first required invalid input that has a validation schema
```

Comment:
I think we can remove the word "required" here. I think we're just finding the first invalid input.

Reply:
Bumping this comment- Per John's comment, I think this would be more correct?
```suggestion
        // Find the first invalid input that has a validation schema
```

Reply:
Correct. Word has now been removed.